Test/ParallelPrimitives/RadixSort.h

```diff
@@ -10,7 +11,14 @@ namespace Oro
 class RadixSort
 {
   public:
-	typedef unsigned int u32;
```

Comment:
Strictly speaking, the size of  `unsigned int` is not fixed	and could be 2 or 4 bytes depends on the target platform. I used the fixed-size definition in `<cstdint>` instead.